cover.py

```diff
@@ -169,6 +171,16 @@ def tx_code(self, code: int):
         wave = self._pi.wave_create()
         self._pi.wave_send_once(wave)
 
+        #for i in range(0, len(wf)):
+        #    _LOGGER.info(
+        #        "waveform %i: %i %i %i",
+        #        i,
+        #        wf[i].gpio_on,
+        #        wf[i].gpio_off,
+        #        wf[i].delay,
+        #    )
```

Comment:
It would be nice to either remove the commented out code, or alternatively log it on an appropriate log level and uncomment it.